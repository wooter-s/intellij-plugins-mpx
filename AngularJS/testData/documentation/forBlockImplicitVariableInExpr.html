@for(item of items) {
  {{$fi<caret>rst}}
}