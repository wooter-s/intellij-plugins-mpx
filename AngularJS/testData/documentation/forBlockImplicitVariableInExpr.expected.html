<div class='definition'><pre>$first:&#32;<span style=""></span><span style="color:#000080;">boolean</span></pre></div>
<div class='content'>
<p>Whether the current row is the first row</p>
</div><table class='sections'>
<tr><td valign='top' class='section'><p>Library:</td><td valign='top'><p>@angular/core@17.0.0-rc.0</td>
</table>